src/Dfc.CourseDirectory.WebV2/README.md

```diff
@@ -22,6 +22,10 @@ For an intro to MediatR in controllers see:
 * https://jonhilton.net/2016/06/06/simplify-your-controllers-with-the-command-pattern-and-mediatr/
 * https://alexlindgren.com/posts/thin-controllers-using-mediatr-with-aspnet-mvc/
 
+### OnOf
+
+For extra compile-time safety we're making heavy use of the [OneOf](https://www.nuget.org/packages/OneOf/) library / pattern.
```

Comment:
In which circumstances are we using int?

Reply:
I'm not really sure :grin:

Reply:
@gunndabad maybe you could answer that one?

Reply:
discussed on slack